src/metabase/email/messages.clj

```diff
@@ -120,8 +120,8 @@
         message-body (stencil/render-file "metabase/email/new_user_invite"
                                           (merge (common-context)
                                                  {:emailType     "new_user_invite"
-                                                  :invitedName   (:first_name invited)
-                                                  :invitorName   (:first_name invitor)
+                                                  :invitedName   (or (:first_name invited) (:email invited))
+                                                  :invitorName   (or (:first_name invitor) (:email invitor))
```

Comment:
If the first name is nil, we can fall back to the email. Wasn't needed before as names _used to be_ required, but aren't anymore.